README.md

```diff
@@ -6,6 +6,8 @@
 
 Simplify virtually executes an app to understand its behavior and then tries to optimize the code so that it behaves identically but is easier for a human to understand. Each optimization type is simple and generic, so it doesn't matter what the specific type of obfuscation is used.
 
+IMPORTANT: Simplify requires Java Development kit (JDK), version 8 prefered.
```

Comment:
The JDK is only required for building, so it's confusing to note here. A better place is the building section.

Reply:
Ok, will move it in the building section as requested.

Reply:
Fixed.